introduction_to_applying_machine_learning/deep_demand_forecasting/deep_demand_forecast.ipynb

```diff
@@ -0,0 +1,3218 @@
+{
```

Comment:
<ol><li>It is unclear if the licensing info applies to fbprophet or PyMeeus?</li><li>I see the PyMeeus license is fine, what about fbprophet?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3729/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
Hi @atqy fbprophet is MIT license. See: https://pypi.org/project/fbprophet/

Reply:
<p>Sounds good, can you make it clearer which package the above license information belong to. Like the below part, I don't know if it belongs to PyMeeus or fbprophet</p><p> </p><p>This package depends on and incorporates or retrieves a third-party software package (such as an open source package) at install-time or build-time or run-time ("External Dependency"). The External Dependency is subject to a license term   that you must accept in order to use this package. If you do not  accept the applicable license term, you should not use this package. We  recommend that you consult your company’s open source approval policy before  proceeding. </p><p>Provided below is the External Dependency and the applicable license  identification as indicated by the documentation associated with the External  Dependency as of Amazon's most recent review. </p><p>THIS INFORMATION IS PROVIDED FOR CONVENIENCE ONLY. AMAZON DOES NOT PROMISE THAT  THE LIST OR THE APPLICABLE TERMS AND CONDITIONS ARE COMPLETE, ACCURATE, OR  UP-TO-DATE, AND AMAZON WILL HAVE NO LIABILITY FOR ANY INACCURACIES. YOU SHOULD  CONSULT THE DOWNLOAD SITES FOR THE EXTERNAL DEPENDENCIES FOR THE MOST COMPLETE  AND UP-TO-DATE LICENSING INFORMATION. </p><p>YOUR USE OF THE EXTERNAL DEPENDENCIES IS AT YOUR SOLE RISK. IN NO EVENT WILL  AMAZON BE LIABLE FOR ANY DAMAGES, INCLUDING WITHOUT LIMITATION ANY DIRECT,  INDIRECT, CONSEQUENTIAL, SPECIAL, INCIDENTAL, OR PUNITIVE DAMAGES (INCLUDING  FOR ANY LOSS OF GOODWILL, BUSINESS INTERRUPTION, LOST PROFITS OR DATA, OR  COMPUTER FAILURE OR MALFUNCTION) ARISING FROM OR RELATING TO THE EXTERNAL  DEPENDENCIES, HOWEVER CAUSED AND REGARDLESS OF THE THEORY OF LIABILITY, EVEN  IF AMAZON HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES. THESE LIMITATIONS  AND DISCLAIMERS APPLY EXCEPT TO THE EXTENT PROHIBITED BY APPLICABLE LAW. </p>